libs/features/smart-scripts/src/sai-search-existing/sai-search-existing.component.html

```diff
@@ -10,10 +10,10 @@
           <label class="col-sm-5 col-md-4 col-lg-3 col-form-label" for="source_type" [translate]="'SOURCE_TYPE'"></label>
           <div class="col-sm-7 col-md-8 col-lg-9">
             <select formControlName="source_type" class="form-control form-control-sm" id="source_type">
-              <option [ngValue]="" [label]=""></option>
               @for (key of SAI_SEARCH_TYPES_KEYS; track key) {
-                <option [ngValue]="key" [label]="SAI_SEARCH_TYPES[key]"></option>
+                <option [ngValue]="key" [label]="`${key} - ${SAI_SEARCH_TYPES[key]}`"></option>
               }
+              <option [ngValue]="" [label]="'All (you need to higher Limit number, to see all Source Types)'"></option>
```

Comment:
not sure about the sentence in parentheses, moreover I would put it in the en.json file for translations
Let's put only `All` the user will understand the limit

Reply:
I will finish the review changes today afternoon and I've a little addition also